paper/paper.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Streamz: A small library to manage continuous streams of data'
+tags:
+  - Stream
+  - Dask
+authors:
+ - name: Matthew Rocklin
+   orcid: ?
+   affiliation: 1
+ - name: Christopher J. Wright
+   orcid: 0000-0003-2522-7028
+   affiliation: 2
+ - name: Julien Lhermitte
+   orcid: 0000-0003-0660-975X
+   affiliation: 5
+ - name: Daniel B. Allan
+   orcid: ?
+   affiliation: 4
+ - name: Simon J. L. Billinge
+   orcid: ?
+   affiliation: 2, 3
+affiliations:
+ - name: Continuum Analytics
+   index: 1
+ - name: Department of Applied Physics and Applied Mathematics, Columbia University
```

Comment:
usually there is a zip code or sthg for affiliation.  Please check journal style.

Reply:
It seems that most don't have a zip code.